### ISSUE TITLE
Critical Game Progression Bug on Planet Erekir: Unable to Advance in Mindustry Storyline

### ISSUE THREAD

**Issue by ilvesBogdan** [Anuken/Mindustry#9349]
### Platforms

Android

### Build

f-droid release build 146

### Issue

I hope this message finds you well. I am writing to report a critical bug that I encountered while playing on Planet Erekir. According to the storyline, the objective was to defeat the enemy using a tank; however, I unintentionally used a turret instead.

After successfully defeating the enemy with the turret, the game did not recognize the completion of the objective, and I am unable to advance further in the storyline. It seems that the system is not acknowledging the alternative method of defeating the enemy.

### Steps to reproduce

We build a turret and destroy a combat unit with it during training

### Save file

[save.zip](https://github.com/Anuken/Mindustry/files/13556988/save.zip)

### (Crash) logs

[logs.zip](https://github.com/Anuken/Mindustry/files/13556984/logs.zip)

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.

**Comment by mahdoosh1** [comment]
If game did not recognize you killing enemies, wait until new enemies spawn
Then you can kill them with tanks.

**Comment by eeve-lyn** [comment]
there's undiscovered terrain you have to explore with tanks

**Comment by ilvesBogdan** [comment]
Hello mahdoosh1 and eeve-lyn,

I trust this message finds you well. I wanted to provide an update regarding the reported issue on Planet Erekir. After further exploration and experimentation, I've successfully resolved the problem that I initially encountered.

It turns out that the game mechanics were functioning as intended. Following your suggestions, I created a tank factory, destroyed the enemy core, and explored undiscovered terrain with my tanks. This allowed me to progress through the storyline without any hindrance.

Thank you once again for your assistance.
